app/models/manageiq/providers/nuage/manager_mixin.rb

```diff
@@ -92,10 +94,10 @@ def verify_api_credentials(options = {})
     with_provider_connection(options) {}
     true
   rescue => err
```

Comment:
Please either don't touch the rescue block if you don't need to 😉 or do it properly, because currently it looks like this 🤢 :

```ruby
rescue => err
	_log.error("Error Class=#{err.class.name}, Message=#{err.message}")
	miq_exception = translate_exception(err)
	raise unless miq_exception

	raise miq_exception
end
```

which could be simplified into:

```ruby
rescue => err
	_log.error("Error Class=#{err.class.name}, Message=#{err.message}")
	raise translate_exception(err)
end
```

Reply:
@miha-plesko i have noticed this block of code was refactored in https://github.com/ManageIQ/manageiq-providers-nuage/pull/40 
I think it is best i revert my change here and wait for mergeing of code from the linked PR. Do you agree?